### ISSUE TITLE
Empty columns in entry_metadata.csv

### ISSUE THREAD

**Issue by Pantagrueliste** [cu-mkp/m-k-manuscript-data#1926]
entry_metadata.csv contains several errors. 

1.  tagnames are not updated `ge_tl` instead of `el_tl` 
2. Columns 'def_tl', and 'ge_tl' are empty 
al_tl     167
bp_tl     116
cn_tl      25
def_tl      0
env_tl    158
m_tl      827
md_tl      61
ms_tl     254
mu_tl       7
pa_tl     186
pl_tl      97
pn_tl      31
pro_tl    169
sn_tl      48
tl_tl     535
tmp_tl    125
wp_tl      50
de_tl       5
ge_tl       0
it_tl       5
la_tl      51
oc_tl      33
po_tl      17

Am I using the wrong version? It was last updated in Nov 27 2020 at 1:16pm

**Comment by njr2128** [comment]
Confirming:
`<def>` is an old, incorrect version of `<df>` (definition) and should be removed from entry-metadata (and should come up empty when searched for)
`<ge>` is an old, incorrect version of `<el>(greek) and should be treated the same

**Comment by gschare** [comment]
> Confirming:
> `<def>` is an old, incorrect version of `<df>` (definition) and should be removed from entry-metadata (and should come up empty when searched for)
> `<ge>` is an old, incorrect version of `(greek) and should be treated the same

The file with the tag names being currently used in `manuscript-object` is outdated.
Confirming that I will update:
- `<ge>` -> `<el>`
- `<def>` -> `<df>`

The tags now being used in `manuscript-object` are:
```py
{
    'animal': 'al',
    'body_part': 'bp',
    'currency': 'cn',
    'definition': 'df',
    'environment': 'env',
    'material': 'm',
    'medical': 'md',
    'measurement': 'ms',
    'music': 'mu',
    'plant': 'pa',
    'place': 'pl',
    'personal_name': 'pn',
    'profession': 'pro',
    'sensory': 'sn',
    'tool': 'tl',
    'time': 'tmp',
    'weapon': 'wp',
    'german': 'de',
    'greek': 'el',
    'italian': 'it',
    'latin': 'la',
    'occitan': 'oc',
    'poitevin': 'po'
}
```

**Comment by njr2128** [comment]
SIDE NOTE: we noticed here that `<fr>` is missing from the list of tags used above. This was a conscious decision at some point as it only appears in one of the versions (tl and NOT in tc and tcn) and that it should not be part of the List of Entries resource on edition640. 
However, we decided that it should be included in the metadata and any other derivate files as part of the dataset of the manuscript. 
For this, see https://github.com/cu-mkp/manuscript-object/issues/63
